### PR TITLE
Truncate labels over 32 characters.

### PR DESCRIPTION
**Pull request by Sid-Data-Universe**

### CODE REVIEW

File: scraping/reddit/model.py [hunk 1]
```diff
@@ -55,7 +55,7 @@ def to_data_entity(cls, content: "RedditContent") -> DataEntity:
             uri=content.url,
             datetime=content.created_at,
             source=DataSource.REDDIT,
-            label=DataLabel(value=content.community),
+            label=DataLabel(value=content.community[:32]),
```
Comment (RusticLuftig) [inline scraping/reddit/model.py:58]: Use our const

File: tests/scraping/reddit/test_model.py [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+            username="user1",
+            communityName="r/looooooooooooooooooooooooongSubreddit",
+            body="Hello world",
+            createdAt=timestamp,
+            dataType=RedditDataType.POST,
+            title="Title text",
+        )
+        entity = RedditContent.to_data_entity(content)
+
+        self.assertEqual(len(entity.label.value), 32)
```
Comment (RusticLuftig) [inline tests/scraping/reddit/test_model.py:23]: Can you also assert the value is what you expect? i.e. that the original communitName is > 32 chars

**Review by RusticLuftig — CHANGES_REQUESTED** [review]

**Review by RusticLuftig — APPROVED** [review]

File: scraping/reddit/model.py (hunk 1 above)
Comment (Sid-Data-Universe) [reply scraping/reddit/model.py]: Done.

File: tests/scraping/reddit/test_model.py (hunk 2 above)
Comment (Sid-Data-Universe) [reply tests/scraping/reddit/test_model.py]: Done.

**Pull request merged**